onnxruntime/hosting/protobuf/predict.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+import "onnx-ml.proto";
+
+package onnxruntime.hosting;
+
+// PredictRequest specifies how inputs are mapped to tensors  
+// and how outputs are filtered before returning to user. 
+message PredictRequest { 
+    reserved 1;
+    map<string, onnx.TensorProto> inputs = 2;
```

Comment:
Add docstring. What is the `string`? I assume the full model input ID?

Reply:
Updated